### PR TITLE
chore: update testnet variables

### LINKED ISSUE #717: ️🛠 - Update Shimmer EVM encoding for production network
### Task description

ShimmerEVM (production) encoding change: 
```
basically we changed the way we identify "sender contracts" on  L1 requests
how does firefly encode this right now?
I'm guessing at this time is a uint32 (4 bytes) as 0s, right?
now just 1 byte (as 0) is needed, that's it
a single 0 byte (means  no contract)
a single byte with value 1, followed by 4 bytes (hname) - (this is the same as it was before, just with the "kind" byte added as a prefix)
a single byte with value 2, followed by 20 bytes (EVM address)
```

We need to align this for production profiles

### Requirements

TBD

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/login/views/SelectProfileView.svelte [hunk 1]
```diff
@@ -56,6 +56,8 @@
             }
             $onboardingProfile = undefined
         }
+        // Ensure there is no active profile set from previous app activity
+        resetActiveProfile()
```
Comment (Tuditi) [inline packages/desktop/views/login/views/SelectProfileView.svelte]: Why is this required?

Comment (Tuditi) [reply packages/desktop/views/login/views/SelectProfileView.svelte]: I removed it from this PR, so we can merge this faster. Feel free to add it back in another PR

**Review by MarkNerdi — APPROVED** [review]

**Review by Tuditi — APPROVED** [review]
Works for me!
